src/main/java/io/deepstream/Record.java

```diff
@@ -244,6 +247,85 @@ public Record set(String path, Object value ) throws DeepstreamRecordDestroyedEx
     }
 
     /**
+     * Set the value for the entire record synchronously and gives acknowledgement
```

Comment:
Is this the actual api we use for set? If it is we should change it to JSONObject or JSONElement since we can't set a string or number using this api